src/tests/global_variables/env_variables_defaults.py

```diff
@@ -230,3 +247,5 @@ class _EnvVariables(DataPool, ABC):
     redfish_machines: EnvVar = EnvVar(
         ["REDFISH_MACHINES"], default=[], loader=lambda machines: re.split(r"\s|,", machines)
     )
+
+    primary_stack = EnvVar(["PRIMARY_STACK"], loader=lambda x: _validate_primary_stack(x), default=env_defaults.DEFAULT_PRIMARY_STACK)
```

Comment:
@linoyaslan , Question .
We have  3 state here:
primary_stack = "None"  -> mean single stack
primary_stack = "ipv4" -> dual stack ipv4 primary
primary_stack = "ipv6" -> dual stack ipv6 primary

if we did not configure env PRIMARY_STACK ,  Are we going to set ipv4 as default ?